cataclysmbnteam/Cataclysm-BN
# fix: separate house22 variants

## Purpose of change
Fix because the two uninhabited house22 variants (after the normal variant) don't have stairs down to the basement below their mapgen.
This results in an inaccessible basement below.
## Describe the solution
Separate the uninhabited house22 variants.
## Describe alternatives you've considered
house22 variants have not been fixed in cdda, so no port.
Put a stair in the variants, but the basement wouldn't fit because it's not empty.
## Testing
1. Check JSON for errors.
2. Start a new game and no errors.
3. Nothing more.
## Additional context
nothing
